spec/services/staff_nomis_checker_spec.rb

```diff
@@ -99,6 +107,10 @@
         describe "and the API is unavailable" do
           let(:offender) { Nomis::NullOffender.new(api_call_successful: false) }
 
+          before do
```

Comment:
What do you think about having a context for when the api is enabled? That way we can avoid having the repetition of having to enable the feature flag